src/colorizer/Dataset.ts

```diff
@@ -223,6 +234,7 @@ export default class Dataset {
     this.frameFiles = manifest.frames;
     this.featureFiles = manifest.features;
     this.outlierFile = manifest.outliers;
+    this.metadata = manifest.metadata;
```

Comment:
Nit/Idea: I wonder if it would be nicer to just not use Partials here and instead call a function that turns the necessarily partial `manifest.metadata` into a complete `this.metadata`.

Reply:
It's not a huge difference for code niceness I think, but I did make this change because it also captures a potential edge case! The new default is for frame dimensions to be 0 (width & height= 0), and the scale bar handling code will now hide the scale bar and skip the calculations so it doesn't divide by 0.

Reply:
I like it.  I feel like Partials make such a weak statement about types, that it's like one level removed from `any`.  This kind of change is generally beneficial.  Less `?`s too.